docs/design/resource-strategy-fit-scheduling.md

```diff
@@ -0,0 +1,95 @@
+# Enhanced NodeResourceFit Plugin
+
+## Summary
+
+The NodeResourcesFit plug-in of native k8s can only adopt a type of strategy for all resources, such as MostRequestedPriority and LeastRequestedPriority. However, in industrial practice, this design does not apply to some scenarios. For example: In AI scenarios, businesses that apply for GPUs prefer to occupy the entire GPU machine first to prevent GPU fragmentation; businesses that apply for CPU & MEM are prioritized and dispersed to non-GPU machines to prevent excessive consumption of CPU & MEM on GPU machines, resulting in real tasks of applying for GPUs. Pending due to insufficient non-GPU resources
+. Therefore, two plugins are extended to solve this common problem.
+
+## Motivation
+
+- GPU tasks take priority over the entire GPU
+- CPU&MEM tasks are distributed to the CPU machine first
+
+## Design Consideration
+
+- The solution is more versatile, not limited to AI clusters or CPU clusters, and not limited to common CPU resources or extended GPU resources.
+
+- Different resource policies can be configured for different cluster types and prioritized in the form of weights.
+
+- Easy to expand
+
+### Goals
+
+- Different types of resources can be configured with different strategies to prioritize them in the form of weights
+
+- Prevent pods that have not applied for scarce resources from being scheduled to nodes with scarce resources.
+
+### Non-Goals
+
+- None.
+
+## Proposal
+
+Extend two plug-ins to meet the above needs
+
+- NodeResourcesFitPlus
+- ScarceResourceAvoidance
```

Comment:
Don’t mention ·ScarceResourceAvoidance· and the SRA strategy here, because there is another document, and the SRA function will not be integrated into your plugin.